### PR TITLE
OSDOCS3020- Added new disk provisioning type for vSphere IPI

### PR DESCRIPTION
**Pull request by bscott-rh**
CP into 4.10
https://issues.redhat.com/browse/OSDOCS-3020

Doc preview: https://deploy-preview-39758--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.html#installation-installer-provisioned-vsphere-config-yaml_installing-vsphere-installer-provisioned-customizations

### CODE REVIEW

**Comment by bscott-rh** [comment]
@ayesha54 @jinyunma Doc work is ready for SME and QA review and approval

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -1267,6 +1267,10 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 |`platform.vsphere.memoryMB`
 |The size of a virtual machine's memory in megabytes.
 |Integer
+
+|`platform.vsphere.disk_type`
```
Comment (jinyunma) [inline modules/installation-configuration-parameters.adoc:1271]: here should be `platform.vsphere.diskType`

Comment (bscott-rh) [reply modules/installation-configuration-parameters.adoc]: Correction has been made

**Comment by jinyunma** [comment]
LGTM

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**